CHANGELOG.md

```diff
@@ -5,6 +5,45 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.7]
+
+### Added
+
+- Secrets are now able to be pushed up during a deploy in the added Secrets
+  view. Secret names can be added to the Configuration, and values supplied
+  individually in the Secrets view. (#2260, #2304, #2305)
+- Newly made Deployments can now be associated with previously deployed content
+  to easily update content on a Connect server. (#2285)
+- Views that require action now show a warning icon when collapsed to assist in
+  discovery (#2245)
+- Credentials are now persisted on OSes without keychains by storing them in the
+  user's home directory under `.connect-credentials`. If using a previous
+  version of the extension with a `.connect-credentials` dotfile remove it when
+  upgrading. (#1831)
+- Files that have previously been deployed that are now deselected for the
+  next deployment show a red `R` icon in the Project Files view (#2299)
+
+### Changed
+
+- When creating a new deployment, open files are listed first with an option to
+  select any file, rather than showing all potential files initially (#2237)
```

Comment:
I'd recommend rewording this (I was confused on the original language)
When creating a new deployment, the user now selects between open entrypoint files along with the option of using the open file dialog to select a file. This removes a recursive search for all entrypoints present within the editor workspace. (#2337)